core/app/[locale]/store-selector/_components/locale-link.tsx

```diff
@@ -0,0 +1,60 @@
+import { Link } from '~/components/link';
+import { LocaleType } from '~/i18n';
+import { cn } from '~/lib/utils';
+
+/**
+ * Custom map of locale to language and region
+ * Temporary solution until we have a better way to include regions for all locales
+ */
+const localeLanguageRegionMap = {
+  da: { language: 'Dansk', region: 'Danmark', flag: 'dk' },
+  en: { language: 'English', region: 'United States', flag: 'us' },
+  'es-419': { language: 'Español', region: 'America Latina', flag: '' },
+  'es-AR': { language: 'Español', region: 'Argentina', flag: 'ar' },
+  'es-CL': { language: 'Español', region: 'Chile', flag: 'cl' },
+  'es-CO': { language: 'Español', region: 'Colombia', flag: 'co' },
+  'es-LA': { language: 'Español', region: 'America Latina', flag: '' },
+  'es-MX': { language: 'Español', region: 'México', flag: 'mx' },
+  'es-PE': { language: 'Español', region: 'Perú', flag: 'pe' },
+  es: { language: 'Español', region: 'España', flag: 'es' },
+  it: { language: 'Italiano', region: 'Italia', flag: 'it' },
+  nl: { language: 'Nederlands', region: 'Nederland', flag: 'nl' },
+  pl: { language: 'Polski', region: 'Polska', flag: 'pl' },
+  pt: { language: 'Português', region: 'Portugal', flag: 'pt' },
+  de: { language: 'Deutsch', region: 'Deutschland', flag: 'de' },
+  fr: { language: 'Français', region: 'France', flag: 'fr' },
+  ja: { language: '日本語', region: '日本', flag: 'jp' },
+  no: { language: 'Norsk', region: 'Norge', flag: 'no' },
+  'pt-BR': { language: 'Português', region: 'Brasil', flag: 'br' },
+  sv: { language: 'Svenska', region: 'Sverige', flag: 'se' },
+};
+
+const getCountryFlagEmoji = (countryCode: string) => {
+  const codePoints = countryCode
+    .toUpperCase()
+    .split('')
+    .map((char) => 127397 + char.charCodeAt(0));
+
+  return String.fromCodePoint(...codePoints);
+};
```

Comment:
I wonder if I should just add the emoji flags instead of using this. 🤔

Reply:
Do you mean adding them to the map? Yeah